pkg/discovery/dtofactory/property/application_properties.go

```diff
@@ -36,6 +37,15 @@ func BuildAppProperties(podNamespace, podName string) []*proto.EntityDTO_EntityP
 	}
 	properties = append(properties, nameProperty)
 
+	containerIndexName := appPropertyNameHostingContainerIndex
+	containerIndexValue := string(index)
```

Comment:
string(index) will return an empty string "".
should use fmt.Sprintf("%d", index)